src/app/home/clubs-list/clubs-list.component.html

```diff
@@ -10,12 +10,12 @@
             </div>
         </div>
     </div>
+    <dx-load-panel
+        message="Loading..."
+        [shading]="true"
+        [showPane]="false"
+        shadingColor="#fff"
+        [position]="{ of: '#clubs'}"
+        [visible]="loadingData">
```

Comment:
I think it's possible to solve the `loadPanel` problem without transfer state usage.